File: src/Port.java
public class Port{
	String MAC,
			IP,
			MTU;
	public int number;
	public Port(String mac,String IP,String MTU){
		this.MAC = mac;
		this.IP = IP;
		this.MTU = MTU;
	}

	public String toString(){
		return ""+number;
	}
}

File: cpp/src/port.cpp
#include "port.hpp"
using namespace std;
/*
port& port::operator=( const port& p )
{
    mac = p.mac;
    ip = p.ip;
    mtu = p.mtu;
    return *this;
}
//*/
string port::toString(){
    stringstream ss;
    ss << this->mac << ","<< this->ip << "," << this->mtu;
    return ss.str();
}
